frontend/src/renderCollaboratorList.jsx

```diff
@@ -6,40 +6,40 @@
 import React, { render } from 'preact-compat';
 
 export default function renderCollaboratorList( container, users, pageType ) {
-	render(<CollaboratorList pageType={ pageType } users={ users } />, container);
+	render( <CollaboratorList pageType={ pageType } users={ users } />, container );
 }
 
-function CollaboratorList ({ users, pageType }) {
-    return (
-        <div>
-	        <div className="text-bold discussion-sidebar-heading" >
-		        { users.length ? `Also on this ${ getPageType( pageType ) }` : 'No other users connected' }
-	        </div>
-	        { users.map( user =>
-		        <Collaborator user={ user } />
-		    ) }
-        </div>
-    );
+function CollaboratorList( { users, pageType } ) {
+	return (
+		<div>
+			<div className="text-bold discussion-sidebar-heading" >
+				{ users.length ? `Also on this ${ getPageType( pageType ) }` : 'No other users connected' }
+			</div>
+			{ users.map( user =>
+				<Collaborator user={ user } key={ user.id } />
+			) }
+		</div>
+	);
+}
 
-	function getPageType( pageType ) {
-		return pageType === 'issues' ? 'issue' : 'pull request';
-	}
+function Collaborator( { user } ) {
+	return (
+		<span key={ user.id } className="css-truncate js-issue-assignees sidebar-assignee">
```

Comment:
If we used `key` property in line **19** do we still need it there? IMO the key property there is redundant.

Reply:
Actually, it should be used on line 19 only. This was a bug before.